### PR TITLE
Plugin to preserve file dates and times from original file after transcode

### LINKED ISSUE #179: Preserve file creation and modification dates after processing
I would like a plugin that would allow me the preserve the creation and last modification dates of a media file after transcoding.  This seems like it should be possible to modify the exiting "rename to 265" plugin that already exists.

### PR DESCRIPTION
**Pull request by WizADSL**
Closes #179

### CODE REVIEW

**Comment by supersnellehenk** [comment]
Could you fix the failing checks? :)

**Comment by WizADSL** [comment]
I'm very sorry to says I'm new at this.  Looking the the output of the check, it seems like the reported errors are not with the file I created.

**Comment by supersnellehenk** [comment]
> I'm very sorry to says I'm new at this. Looking the the output of the check, it seems like the reported errors are not with the file I created.

You're absolutely right. I assumed it only checked this file, but it doesn't. So you can disregard my old message about that.

**Review by supersnellehenk — APPROVED** [review]

File: Community/Tdarr_Plugin_z80t_keep_original_date.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+		var extensions = inputs.extensions;
+		if (extensions == undefined || extensions.trim() === '') {
+			extensions = librarySettings.containerFilter;
+		}
+		extensions = extensions.split(',');
+
+		await new Promise(resolve => setTimeout(resolve, 5000));
+		var response = await getFileData(file._id, extensions, inputs.server);
+
+		//log('Got response for ' + file._id);
```
Comment (supersnellehenk) [inline Community/Tdarr_Plugin_z80t_keep_original_date.js:84]: Could you remove all the commented out lines of code though?

Comment (WizADSL) [reply Community/Tdarr_Plugin_z80t_keep_original_date.js]: Done.

**Comment by HaveAGitGat** [comment]
@WizADSL I've fixed the lint errors. Please could you double check the plugin is still working as you like :), thanks

**Pull request merged**